packages/react-components/src/Status/index.tsx

```diff
@@ -144,8 +145,24 @@ function Status ({ className, stqueue = [], txqueue = [] }: Props): React.ReactE
     return null;
   }
 
+  const _onDismiss = (): void => {
+    allst.map(s => s.removeItem());
```

Comment:
(nitpick) Prefer the following form 

```js
allst.map((s): void => s.removeItem())`
```

1. no naked params, i.e. `(s)`
2. it is strange, but linting should complain here - the `: void` should be picked up as missing

Reply:
Probably because of the missing parentheses 🤔